packages/next/cli/next-build.ts

```diff
@@ -8,17 +8,23 @@ import build from '../build'
 import { printAndExit } from '../server/lib/utils'
 
 const nextBuild: cliCommand = (argv) => {
-  const args = arg(
-    {
-      // Types
-      '--help': Boolean,
-      '--profile': Boolean,
-      // Aliases
-      '-h': '--help',
-    },
-    { argv }
-  )
+  const validArgs: arg.Spec = {
+    // Types
+    '--help': Boolean,
+    '--profile': Boolean,
+    // Aliases
+    '-h': '--help',
+  }
 
+  let args: arg.Result<arg.Spec>
+  try {
+    args = arg(validArgs, { argv })
+  } catch (error) {
+    if (error.code === 'ARG_UNKNOWN_OPTION') {
+      printAndExit(error.message, 1)
```

Comment:
Probably not strictly necessary, but I'd return here for readability purposes
```suggestion
      return printAndExit(error.message, 1)
```

Reply:
Sure! Let me push it

Reply:
Pushed in 6bdca04